src/ocean/util/serialize/contiguous/package_.d

```diff
@@ -734,10 +734,10 @@ unittest
 {
     static struct CS
     {
-        cstring s;
+        mstring s;
```

Comment:
No, this will be a breaking change in `v4.x.x` - such struct will no longer deserialize (it was never intended to). I wonder though how will it affect bidding apps as I presume this test was added by @mathias-lang-sociomantic originally.